expressServer/scripts/generateUsernames.js

```diff
@@ -1,7 +1,7 @@
 const mongoose = require('mongoose');
 const { uniqueNamesGenerator, adjectives, colors, animals } = require('unique-names-generator');
 const dotenv = require('dotenv');
-
+const { connectDB, closeDB } = require('../libraries/database.js');
```

Comment:
You forgot to centralize the following functionalities as well: 
- mongoose.connect
- userSchema
- mongoose.model
They are still declared everywhere instead of using them from a centralized point. 

For example, all your work you have declared the following code in your other scripts but it's not updated in the rest: 

```
const userSchema = new mongoose.Schema({
    username: { type: String, required: true, unique: true },
    password: { type: String, required: false },
    hash: { type: String, required: false },
    salt: { type: String, required: false }
});
```